mgmt/RecordsConfig.cc

```diff
@@ -103,10 +103,10 @@ static const RecordElement RecordsConfig[] =
   ,
   {RECT_CONFIG, "proxy.config.output.logfile.rolling_size_mb", RECD_INT, "100", RECU_DYNAMIC, RR_NULL, RECC_STR, "^0*[1-9][0-9]*$", RECA_NULL}
   ,
-  {RECT_CONFIG, "proxy.config.snapshot_dir", RECD_STRING, "snapshots", RECU_NULL, RR_REQUIRED, RECC_NULL, NULL, RECA_NULL}
+  {RECT_CONFIG, "proxy.config.snapshot_dir", RECD_STRING, "snapshots", RECU_DYNAMIC, RR_REQUIRED, RECC_NULL, NULL, RECA_NULL}
```

Comment:
Yes it is a borderline case. Take a look at ``FileManager::FileManager()``. If we remove the ``managedDir`` member variable, and use ``RecConfigReadSnapshotDir()`` in ``FileManager::WalkSnaps``, then it would be correctly consumed in a dynamic way too.